src/chrome/content/rules/Your-Mailing-List-Provider.xml

```diff
@@ -1,24 +0,0 @@
-<!--
-	Nonfunctional domains:
-
-		- img.ymlp.com *
-
-	* 404, valid cert
-
--->
-<ruleset name="Your Mailing List Provider">
-	<target host="ymlp.com" />
-	<target host="chiclet.ymlp.com" />
-	<target host="www.ymlp.com" />
-	<target host="yourmailinglistprovider.com" />
-	<target host="www.yourmailinglistprovider.com" />
-
-	<securecookie host="^(?:chiclet|www)\.ymlp\.com$" name=".+" />
-
-	<rule from="^(?:https://|http://(?:www\.)?)ymlp\.com/"
-		to="https://www.ymlp.com/" />
-	<rule from="^http://chiclet\.ymlp\.com/"
-		to="https://chiclet.ymlp.com/" />
-	<rule from="^http://(?:www\.)?yourmailinglistprovider\.com/"
```

Comment:
I think we should keep this rule.
See also mismatching `chiclet.yourmailinglistprovider.com`.